### DISCUSSION TITLE
How to show only 1 separator axis at 0?

### DISCUSSION THREAD

**Discussion by danipazos** [Live-Charts/LiveCharts2#714] [Q&A] [answered]
Hi,

Is it possible to show a separator x-axis only for the 0 value in a series that goes from -100 to 100?

Something like this:
![barchart](https://user-images.githubusercontent.com/7761346/197840528-76543b1e-c638-4ed7-a740-5f8658ae0136.png)

Thanks!

**Comment by danipazosdev** [comment, marked as answer]
It took me a while but it was as easy as adding a RectangularSection to my chart and setting ShowSeparatorLines to false.
